config/awesome/surreal/configuration/secrets.lua

```diff
@@ -39,7 +39,24 @@ return {
 
 		key           = '',
 		city_id       = '',
-		units         = 'metric'
+		units         = 'metric',
+		auto_geoinfo  = true
 
+	},
+
+	geoinfo = {
+		ipstack_key		= "",
+		proxy_for_check_ip = false,
```

Comment:
Finally, got a time to review this! Thank you for this as always! Anyway I have questions. 

Is the `ipstack_key` required or is it okay to just leave it empty? And please explain the other configurations like proxy_for_check_ip and etc. What do they do?

Also why are the items under the ***-- items which do not change*** in secrets.lua? Isn't the geoinfo module's the one that will set them?